lib/smack/Prelude.cpp

```diff
@@ -0,0 +1,526 @@
+#include "smack/Prelude.h"
+#include "smack/Naming.h"
+#include "smack/SmackOptions.h"
+#include "smack/Regions.h"
+
+namespace smack {
+
+const std::vector<unsigned> INTEGER_SIZES = {1, 5, 6, 8, 16, 24, 32, 40, 48,
```

Comment:
Is there a particular reason that `INTEGER_SIZES` is a global variable? It appears to me that it's only used in one function.